Snapper/Manager.py

```diff
@@ -0,0 +1,74 @@
+from Snapper.Snapper import Snapper
+from Snapper.AssociationMatrix import AssociationMatrix
+from Relationship.Variable import Variable
+from Relationship.SpearframeRelationship import SpearframeRelationship
+
+
+class Manager:
+
+    def __init__(self):
+        self.sensors = []
+        self.route_map = {}
+        self.variables = []
+        self.snapper = Snapper(self)
+        self.matrix = AssociationMatrix()
+
+    def add_sensor(self, sensor):
```

Comment:
The Sensor has a Snapper attribute to use as a callback for when data is published, but the manager owns the Snapper that needs to be attached to the Sensor. It makes more sense if the Sensor has a `set_snapper_callback()` method that the manager can use in the `add_sensor` function so as to avoid the awkward `Sensor(manager.snapper)` construction. Other reviewers should probably weigh in on this @EliCDavis